CMakeLists.txt

```diff
@@ -160,3 +169,15 @@ if(BUILD_CRTM)
                    "-DCMAKE_INSTALL_PREFIX=${CMAKE_INSTALL_PREFIX}/crtm"
     )
 endif()
+
+if(BUILD_POST)
+  ExternalProject_Add(post
+    GIT_REPOSITORY "https://github.com/aerorahul/emc_post"
+    GIT_TAG        "feature/cmakepkg"
+    PREFIX         "${CMAKE_BINARY_DIR}/post"
+    CMAKE_ARGS     "-DCMAKE_PREFIX_PATH=${CMAKE_INSTALL_PREFIX}"
+                   "-DCMAKE_INSTALL_PREFIX=${CMAKE_INSTALL_PREFIX}/post"
```

Comment:
Post processor library (.a file, cmake package configuration files and imported target) is named **nceppost**. I suggest the install directory be also named ``nceppost`` instead of just ``post``. It will make scripting much easier in the future if everything is consistently named. Also for consistency ``PREFIX`` can be changed too.

Reply:
done.